witchcraft-server/src/logging/logger/rolling_file.rs

```diff
@@ -80,7 +89,7 @@ impl RollingFileAppender {
         let len = file.metadata().await.map_err(Error::internal_safe)?.len();
 
         let archive_locator = ArchiveLocator::new(name);
-        let date = Utc::now().date();
+        let date = Utc::now().date_naive();
```

Comment:
Why the change to `NaiveDate`?

Reply:
`Date` is now deprecated: https://docs.rs/chrono/latest/chrono/struct.Date.html